### ISSUE TITLE
Syntax highlighting contrasts in docs

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#567]
[We currently ignore `.chroma` from pa11y](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/8145088d68168d39dc58f849bac466e494918b35/build/.pa11yci.json) which feels weird. Should try to get a decent and accessible color palette for code blocks.

Should be backported to v4, however we should wait for migration from Jekyll to Hugo on Bootstrap side.
